Transcripts/Episode1.md

```diff
@@ -0,0 +1,84 @@
+# Episode 1
+### [Youtube Link](https://www.youtube.com/watch?v=S2mK6KFdv0I&list=PLPV2KyIb3jR4KLGCCAciWQ5qHudKtYeP7)
+### [Notion](https://www.notion.so/gamedevmcgill/Dissection-of-Brackey-s-RPG-25c5b38888d840a5b5da528644c5a9ea#7c5b0616c1d942789e4f71fc1fbb1712)
+
+
```

Comment:
**unity setup** (<5 mins)
Review what is a scene, game object, component
Quickly show the scene vs game view, hierarchy window, folder setup